connectors/byoc.py

```diff
@@ -125,7 +125,10 @@ async def update_filtering_validation(
             retry_on_conflict=RETRY_ON_CONFLICT,
         )
 
-    def build_docs_query(self, native_service_types=None, connectors_ids=None):
+    @staticmethod
```

Comment:
why a staticmethod? if it's not using any of the class variables, make this a seperate function